### PR TITLE
feat: Explore Patterns page

### PR DESCRIPTION
**Pull request by DafyddLlyr**
image

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -4,14 +4,14 @@ module.exports = {
...
-        stewardship: "#9FF876",
-        use: "#FEFC41",
+        rent: 'rgb(var(--color-rent) / <alpha-value>)',
+        transfer: 'rgb(var(--color-transfer) / <alpha-value>)',
+        administration: 'rgb(var(--color-administration) / <alpha-value>)',
+        eligibility: 'rgb(var(--color-eligibility) / <alpha-value>)',
+        security: 'rgb(var(--color-security) / <alpha-value>)',
+        develop: 'rgb(var(--color-develop) / <alpha-value>)',
+        stewardship: 'rgb(var(--color-stewardship) / <alpha-value>)',
+        use: 'rgb(var(--color-use) / <alpha-value>)',
```
Comment (jessicamcinchak) [inline tailwind.config.js:14]: just making sure i understand this: this change now gives us option to dynamically pass an opacity value? nice!

File: components/layout/PatternsLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ )
+}
+
+const PatternItem = (props: { pattern: Pattern, patternClassName: string | undefined, reverse?: boolean, highestCount: number | undefined }) => {
+  const { pattern, patternClassName, reverse, highestCount } = props
+
+  return (
+    <div className={`flex mb-4 ${reverse ? "flex-row-reverse" : ""}`}>
+      <div className={clsx(`w-1/2 ${patternClassLookup[patternClassName!]} bg-opacity-40 flex items-center justify-center`)}>
+        <LogoGithub size={32} className="w-1/4 flex-center"/>
```
Comment (jessicamcinchak) [inline components/layout/PatternsLayout.tsx:72]: i hadn't really registered from the figma designs that these patterns were going to link to github - are they on github already (in a repo? as a gist?)? that might be a notion task ticket we're missing!

**Review by jessicamcinchak — APPROVED** [review]
looks good! agree queries are slow on netlify, but totally makes sense to address later / i think it's effecting more than just this page.

File: tailwind.config.js (hunk 1 above)
Comment (DafyddLlyr) [reply tailwind.config.js:14]: https://tailwindcss.com/docs/customizing-colors#using-css-variables

Yep, defining these colours via their RGB colours allow them to behave more like Tailwind colours should 👌

File: components/layout/PatternsLayout.tsx (hunk 2 above)
Comment (DafyddLlyr) [reply components/layout/PatternsLayout.tsx]: Sorry! This is me being a bit too quick/dirty maybe 😅 

It's just a placeholder logo for the pattern logos which are to come - I just copy/pasted an icon from the footer here.

Comment (jessicamcinchak) [reply components/layout/PatternsLayout.tsx]: ha no worries! open cirlcle or simple geometric shape would be a nice swap if easy, but no worries if not! octocat just has such specific meanings to me :octocat:

File: components/layout/PatternsLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <div className={`flex mb-4 ${reverse ? "flex-row-reverse" : ""}`}>
+      <div className={clsx(`w-1/2 ${patternClassLookup[patternClassName!]} bg-opacity-40 flex items-center justify-center`)}>
+        <Hotel size={32} className="w-1/4 flex-center" />
+        <div className="p-4 w-3/4">
+          <p className="text-lg">{pattern.name}</p>
+          <p className="text-xs">{pattern.description}</p>
+        </div>
+      </div>
+      <div className="w-1/2 grid" style={{ gridTemplateColumns: `repeat(${highestCount}, minmax(0, 1fr))`, direction: reverse ? "rtl" : "ltr" }}>
+        <div className={clsx(`${patternClassLookup[patternClassName!]} bg-opacity-20 flex justify-center flex-col py-2 ${reverse ? "pr-4" : "pl-4"}`)} style={{ gridColumn: `span ${pattern.entryCount}` }}>
```
Comment (jessicamcinchak) [inline components/layout/PatternsLayout.tsx:78]: also: clever uses of `reverse` here! i was fiddling with something similar for the charts for awhile but never quite landed on it - definitely going to come back and reference this when i clean up chart labels :slightly_smiling_face:

**Pull request merged**